### PR TITLE
OSDOCS-4658: Cluster to compute machines

### PR DESCRIPTION
**Pull request by kelbrown20**
**For Versions:** 4.12+
**Issue:** [OSDOCS-4658](https://issues.redhat.com//browse/OSDOCS-4658)

**Description:** Update wording on "multi-architecture clusters" to "multi-architecture compute machines" to clearer description of the feature

**Preview:** 
[Post-installation configuration -> Configuring multi-architecture compute machines on a OpenShift Container Platform cluster](https://53552--docspreview.netlify.app/openshift-enterprise/latest/post_installation_configuration/multi-architecture-configuration.html)

**QE review:**
- [x] QE has approved this change.

**Additional info:**
4.11 PR #53558

### CODE REVIEW

**Comment by tmalove** [comment]
@kelbrown20 I looked in the ISG and the OpenShift docs glossary and did not see guidance on using 'a' instead of 'an' to precede 'OpenShift'. I also saw many other instances where 'a' was used, so I reversed my comment about that...looks good otherwise!

File: modules/multi-architecture-upgrade-mirrors.adoc [hunk 1]
```diff
@@ -5,9 +5,9 @@
 :_content-type: PROCEDURE
 [id="multi-architecture-upgrade-mirrors_{context}"]
 
-= Upgrading your multi-architecture cluster
+= Upgrading your multi-architecture compute machines
```
Comment (Prashanth684) [inline modules/multi-architecture-upgrade-mirrors.adoc:8]: this should be - "Upgrading a cluster with multi-architecture compute machines"

File: modules/multi-architecture-upgrade-mirrors.adoc [hunk 2]
```diff
@@ -5,9 +5,9 @@
 :_content-type: PROCEDURE
 [id="multi-architecture-upgrade-mirrors_{context}"]
 
-= Upgrading your multi-architecture cluster
+= Upgrading your multi-architecture compute machines 
 
-You must perform an explicit upgrade command to upgrade your existing cluster to a multi-architecture cluster.
+You must perform an explicit upgrade command to upgrade your existing cluster to a cluster with multi-architecture compute machines.
```
Comment (Prashanth684) [inline modules/multi-architecture-upgrade-mirrors.adoc:11]: should be - "You must perform an explicit upgrade command to upgrade your existing cluster's payload to a payload which supports multi-architecture compute machines"

File: post_installation_configuration/multi-architecture-configuration.adoc [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
+The multi-architecture compute machines Technology Preview feature has limited usability with installing, upgrading, and running payloads. 
 ====
 
-The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This will add `arm64` worker nodes to your multi-architecture cluster and deploy the desired amount of ARM64 virtual machines (VM). This section also shows how to upgrade your existing cluster to a multi-architecture cluster. Multi-architecture clusters are only available on Azure installer-provisioned infrastructures with `x86_64` control planes. 
+The following procedures explain how to generate an `arm64` boot image and create an Azure compute machine set with the `arm64` boot image. This adds `arm64` compute nodes to your cluster and deploys the desired amount of `arm64` virtual machines (VM). This section also shows how to upgrade your existing cluster to a cluster with multi-architecture compute machines. Clusters with multi-architecture compute machines are only available on Azure installer-provisioned infrastructures with `x86_64` control plane machines.
```
Comment (Prashanth684) [inline post_installation_configuration/multi-architecture-configuration.adoc:17]: should be: "This section also shows how to upgrade your existing cluster to a cluster which supports multi-architecture compute machines"

File: post_installation_configuration/multi-architecture-configuration.adoc [hunk 4]
```diff
@@ -1,21 +1,21 @@
...
 :context: multi-architecture-configuration
 [id="post-install-multi-architecture-configuration"]
-= Configuring a multi-architecture cluster
+= Configuring multi-architecture compute machines on an {product-title} cluster 
 include::_attributes/common-attributes.adoc[]
 
 toc::[]
 
-A multi-architecture cluster is a cluster that supports worker machines with different architectures. You can deploy a multi-architecture cluster by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
+An {product-title} cluster with multi-architecture compute machines is a cluster that supports compute machines with different architectures. You can deploy a cluster with multi-architecture compute machines by creating an Azure installer-provisioned cluster using the multi-architecture installer binary. For Azure installation, see xref:../installing/installing_azure/installing-azure-customizations.adoc[Installing a cluster on Azure with customizations].
```
Comment (Prashanth684) [inline post_installation_configuration/multi-architecture-configuration.adoc:9]: "compute machines of different architectures" ?

**Comment by jeana-redhat** [comment]
> @kelbrown20 I looked in the ISG and the OpenShift docs glossary and did not see guidance on using 'a' instead of 'an' to precede 'OpenShift'. I also saw many other instances where 'a' was used, so I reversed my comment about that...looks good otherwise!

Happened to see this comment during merge review - should be "an" before a vowel sound per ISG. Maybe we should sweep the repo for "a OpenShift" and "a {product-title}" :thinking:  But this one should be fixed before merging :)

**Comment by kelbrown20** [comment]
@jeana-redhat Good eye! Thank you Jeana! I know I fixed that in the other one, and I just updated this one 👍

**Pull request merged**